android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java

```diff
@@ -410,6 +413,15 @@ public void onPageLoadFinished(final Tab tab, GURL url) {
                         checkForTooltip(tab);
                     }
                 }
+
+                String countryCode = Locale.getDefault().getCountry();
+                if (countryCode.equals(BraveConstants.INDIA_COUNTRY_CODE)
+                        && url.domainIs("youtube.com")
```

Comment:
Can we use constant for this ? if we need to change domain later, we would be able to do it easily